rabbitinahat/src/main/java/org/ohdsi/rabbitInAHat/DetailsPanel.java

```diff
@@ -315,14 +327,25 @@ public void removeUpdate(DocumentEvent e) {
 	private class FieldPanel extends JPanel implements DocumentListener {
 
 		private static final long	serialVersionUID	= -4393026616049677944L;
-		private JLabel				nameLabel			= new JLabel("");
-		private JLabel				rowCountLabel		= new JLabel("");
-		private DescriptionTextArea description			= new DescriptionTextArea ("");
-		private SimpleTableModel	valueTable			= new SimpleTableModel("Value", "Frequency", "Fraction");
-		private JTextArea			commentsArea		= new JTextArea();
-		private Field				field;
+		JLabel				nameLabel;

```

Comment:
The outlining of this does not seem correct. Within a sentence, please use spaces to do outlining, or forego outlining altogether.